Source/Krypton Components/Krypton.Ribbon/Palette/RibbonAppButton.cs

```diff
@@ -437,7 +437,7 @@ public bool FormCloseBoxVisible
                     if (_ribbon.CaptionArea != null)
                     {
                         _ribbon.CaptionArea.KryptonForm!.CloseBox = value;
-                        _ribbon.CaptionArea.PerformFormChromeCheck();
+                        _ribbon.CaptionArea?.PerformFormChromeCheck();
```

Comment:
Why was this change necessary, as the `CaptionArea` is used on the previous line "Without" the null check.

Reply:
Actually the "KryptonForm" in line 439 should be checked for null at that location, not the CaptionArea as the outer "if" already assures that.

Reply:
Undone, but what's causing #1308?

Reply:
> Undone, but what's causing #1308?

Exactly what I wrote in my above comment and in the issue report:
**_ribbon.CaptionArea.KryptonForm** is null, thus needs to be checked for null before assignment